modules/integration/src/test/scala/scala/cli/integration/CompileTestDefinitions.scala

```diff
@@ -49,81 +49,81 @@ abstract class CompileTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
-  test("test scope") {
-    val inputs = TestInputs(
-      Seq(
-        os.rel / "Main.scala" ->
-          """using lib "com.lihaoyi::utest:0.7.10"
-            |
-            |object Main {
-            |  val err = utest.compileError("pprint.log(2)")
-            |  def message = "Hello from " + "tests"
-            |  def main(args: Array[String]): Unit = {
-            |    println(message)
-            |    println(err)
-            |  }
-            |}
-            |""".stripMargin,
-        os.rel / "Tests.scala" ->
-          """using lib "com.lihaoyi::pprint:0.6.6"
-            |using target test
-            |
-            |import utest._
-            |
-            |object Tests extends TestSuite {
-            |  val tests = Tests {
-            |    test("message") {
-            |      assert(Main.message.startsWith("Hello"))
-            |    }
-            |  }
-            |}
-            |""".stripMargin
-      )
-    )
-    inputs.fromRoot { root =>
-      os.proc(TestUtil.cli, "compile", extraOptions, ".").call(cwd = root)
-    }
-  }
+//  test("test scope") {
```

Comment:
Why is this disabled?

Reply:
In latest commit all tests are re-enabled and re-added